src/components/atoms/NotFound/NotFound.scss

```diff
@@ -0,0 +1,55 @@
+@import "scss/constants";
+
+$message-container-width: 28rem;
+
+.NotFound {
+  height: calc(100% - #{$reserved-page-height});
+  width: 100%;
+  margin: 0;
+  padding: 0;
+
+  display: grid;
+  place-items: center;
+
+  background-color: $black;
+  background-image: url(/assets/images/starfield.jpg);
+  background-size: cover;
+  background-position: center;
+
+  &--full-screen {
+    position: fixed;
+    width: 100vw;
+    height: 100vh;
+    border-radius: 0 !important;
```

Comment:
Why do you need `!important` which style are you overriding?

Reply:
Just checked, it appears it was only an issue for BM branch, will remove it here.

Reply:
done @ c94fdeb6b0b69f3582b9698dc433f36526269a87